src/components/header/header.tsx

```diff
@@ -20,8 +22,15 @@ const { Text } = Typography;
 const HeaderOfApp = () => {
   const location = useLocation();
   const token = useAppSelector((state) => state.auth.userData?.token);
+  const langFromStorage = localStorage.getItem('lang');
 
   const [open, setOpen] = useState<boolean>(false);
+  const [lang, setLang] = useState<string>(langFromStorage ? langFromStorage : 'en');
```

Comment:
```suggestion
  const [lang, setLang] = useState<string>(langFromStorage ?? 'en');
```
тоже можно упростить